modules/ROOT/pages/salesforce/salesforce-connector-100-upgrade-migrate.adoc

```diff
@@ -0,0 +1,214 @@
+= Salesforce - Upgrade and Migrate - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+Upgrade Anypoint Connector for Salesforce (Salesforce Connector) to version 10.0.0.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|9.8.x |10.0.0
+|===
+
+== Changes in This Release
+
+This release contains the following changes:
+
+* Removed deprecated connection types
+* Made the following changes to some operations:
+ ** Merged multiple operations into one
+ ** Renamed some operations and parameters
+ ** Changed some operation return types
+* Redesigned the Replay ID functionality
+* Changed the Mule TLS Context usage
+* Improved the error types
+* Removed the dependency on the Dozer library
+
+=== Changes in Connection Types
+
+This release contains the following changes to connection types:
+
+* Removed the following connection types:
+** Username Password
+** Basic Username Password
+* Added connection pool functionality to improve multi-threading support. With this functionality, each request sent to the API has its own headers and read timeout, which prevents the operation headers from being overridden with the wrong values when operations are executed simultaneously.
+* Added a new *Security* tab for configuring TLS.
+* Reordered the parameters in a more intuitive way.
+* Moved the Read timeout parameter from the provider configuration to the connector configuration.
+* Removed the following parameters from the connector configuration: (*George, can we order these alphabetically, or are they ordered this way for a reason?*)
```

Comment:
Yes, these ones can be ordered alphabetically. I've put them in this order because this is their order in Salesforce configuration.